ephios/settings.py

```diff
@@ -316,6 +321,17 @@ def GET_SITE_URL():
     return site_url
 
 
+def GET_USERCONTENT_URL():
+    return MEDIA_URL
+
+
+def GET_USERCONTENT_QUOTA():
+    """Returns a tuple (used, free) of the user content quota in bytes"""
+    used = sum(p.stat().st_size for p in Path(MEDIA_ROOT).rglob("*"))
+    free = shutil.disk_usage(MEDIA_ROOT).free
```

Comment:
I still think we should have something like
```suggestion
    quota = env.int("MEDIA_FILES_DISK_QUOTA", default=-512 * 1024 * 1024)  # leave 512 MB free by default
    disk_free = shutil.disk_usage(MEDIA_ROOT).free
    free = (disk_free + quota) if quota < 0 else min(disk_free, quota - used)
```

Reply:
I'm not sure if we need the complicated setup with positive and negative values, I think that is rather hard to understand. I'd prefer just setting an absolute amount that is available (as we will do in SaaS) and then do `min(disk_free, quota - used)`

Reply:
sure :)